src/app/room/room.component.html

```diff
@@ -1,15 +1,27 @@
 <copy-link></copy-link>
 
 <ng-container *ngIf="currentPlayer$ | async as storePlayer">
-  <div>Store Player information:</div>
+  <div><b>Store:</b></div>
   <div *ngIf="storePlayer">
-    <div>Id: {{storePlayer.id}}</div>
+    <div>Player Id: {{storePlayer.playerId}}</div>
+    <div>PlayerInRoom Id: {{storePlayer.playerInRoomId}}</div>
+  </div>
+</ng-container>
+
+<ng-container *ngIf="firebasePlayerInRoom$ | async as playerInRoom">
```

Comment:
lovely 👍

Reply:
I am not quite sure what you mean. thanks? :)